internal/deploys/serviceInstance_test.go

```diff
@@ -120,29 +112,17 @@ func (suite *ServiceInstanceIntegrationTestSuite) TestCreateServiceInstance() {
 		result, err := suite.app.serviceInstances.CreateNew(newServiceInstanceReq)
 		suite.Require().NoError(err)
 
-		suite.Assert().Equal(newServiceInstanceReq.ServiceName, result.Service.Name)
+		suite.Assert().NotEqual(0, result.ServiceID)
```

Comment:
nothing wrong here, just me expressing my annoyance that golang forces us to do this. I'm very thankful that no SQL databases start with ID 0.